### PR TITLE
Update tooltips and UI for each filter in Data Processing

### PR DESCRIPTION
**Pull request by marcellp**
# Background
#### Link to issue
https://biomage.atlassian.net/browse/BIOMAGE-704

#### Link to staging deployment URL
https://ui-91go8oejz3b1ryknh1d86wzo7o.scp-staging.biomage.net/

#### Links to any Pull Requests related to this
N/A

#### Anything else the reviewers should know about the changes here
Also fixed some usability problems regarding responsiveness of plots and the data processing page in general by using `flex` layouts.

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [x] Unit tests written
- [x] Tested locally with Inframock
- [x] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [x] Relevant Github READMEs updated
- [x] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

**Comment by aerlaut** [comment]
Looks good, but I prefer the loading bar just beside the Next / Prev button. That's just my preference though,

**Comment by marcellp** [comment]
> Looks good, but I prefer the loading bar just beside the Next / Prev button. That's just my preference though,

Same, but it would make it substantially more difficult to properly make things responsive. I could do it manually but don't want to mess around with the header right now. I'll stage it and see how it looks.

**Comment by marcellp** [comment]
@aerlaut -- Fixed.

**Comment by aerlaut** [comment]
Point no. 4. in the ticket asked for the dropdown to be removed from the Mitochondrial Content Filter. Is this going to be implemented?

**Comment by marcellp** [comment]
I misunderstood, I thought it was only the other mock options that need to be removed. Pushing a fix.

File: src/components/data-processing/MitochondrialContent/CalculationConfig.jsx [hunk 1]
```diff
@@ -72,8 +72,6 @@ const CalculationConfig = (props) => {
 
   const filtering = false;
 
-  const activeMethod = config.method;
```
Comment (aerlaut) [inline src/components/data-processing/MitochondrialContent/CalculationConfig.jsx]: I think you shouldn't remove this, just replace it's value with `absolute_threshold`

**Review by cosa65 — APPROVED** [review]

**Pull request merged**